API.md

```diff
@@ -475,7 +475,7 @@ expression.
 #### Parameters
 
 -   `name` **[string][121]** 
--   `value` **([string][121] \| [RegExp][132])** 
+-   `value` **([RegExp][132] \| any)** 
```

Comment:
this file is automatically generated from the doc comments. make sure to change the the type in the doc comment, and then regenerate it, instead of only changing it here :)

Reply:
Oops. :slightly_smiling_face: 
Updated